test/integration/custom_executable_test.rb

```diff
@@ -35,6 +35,19 @@ class CustomExecutableTest < Minitest::Test
         assert_match(/1 offense detected/, captured_output)
       end
 
+      test "'packwerk check' with violations accounts for custom inflections, fails and displays violations" do
+        Tempfile.create(["timeline_comment", ".rb"], timeline_path("app", "models")) do |file|
+          # payment_details is an uncountable inflection
```

Comment:
I added this comment because it may not be clear to someone who is just reading the code. You would have to know that `payment_details` in `custom_inflections.yml` is an "uncountable", which means it should not be inflected when found in a constant association. 

Without the fix for this bug, this `check` run would pass because Packwerk is unable to resolve the constant and treats it as a non-violation. In reality, `payment_details` exists as a constant, which makes it a legitimate violation.

Reply:
This is good information which deserves a promotion into the commit message 😄